src/main/resources/templates/about.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>About - Common Workflow Language Viewer</title>
+    <link rel="stylesheet" th:href="@{/bower_components/bootstrap/dist/css/bootstrap.min.css}" href="../static/bower_components/bootstrap/dist/css/bootstrap.min.css" />
+    <link rel="stylesheet" th:href="@{/bower_components/highlightjs/styles/github.css}" href="../static/bower_components/highlightjs/styles/default.css" />
+    <link rel="stylesheet" type="text/css" th:href="@{/css/main-20170616.css}" href="../static/css/main-20170616.css" />
+</head>
+<body>
+
+<nav th:replace="fragments/header :: navbar"></nav>
+
+<div class="container">
+    <div class="row">
+        <div class="col-md-12" role="main" id="main">
+            <h1>About</h1>
+            <div style="position:relative;float:right;height:0;padding-bottom:30%;width:50%;margin-left:20px;">
+                <iframe src="https://www.youtube-nocookie.com/embed/_yjhVTmvxLU?rel=0?ecver=2" style="position:absolute;width:100%;height:100%;left:0" allowfullscreen="true"></iframe>
+            </div>
+
+            <p>CWL Viewer is a richly featured web visualisation suite for workflows written in the
+                <a href="http://www.commonwl.org/">Common Workflow Language</a> with an aim of facilitating sharing,
+                understanding and discovery as well as encouraging best practices when writing workflows and their
+                tooling.</p>
+
+            <p>Cite as: <code><a href="https://doi.org/10.7490/f1000research.1114375.1">10.7490/f1000research.1114375.1</a></code></p>
```

Comment:
"Please". Consider following the Crossref DOI display guidelines: https://www.crossref.org/display-guidelines/